### PR TITLE
Add notes about support of Collectd Protocol and fix url

### PR DESCRIPTION
**Pull request by bmillemathias**
Add a section about collectd protocol support
Add a small guide about collectd configuration
Fix url to config.sample.toml

### CODE REVIEW

File: source/docs/v0.8/api/reading_and_writing_data.md [hunk 1]
```diff
@@ -80,11 +80,17 @@ milliseconds. This holds true for both reading and writing data.
...
+
+Note that you need to install file types.db, you can find a version of this file in the GitHub repository of [Collectd](https://github.com/collectd/collectd/blob/master/src/types.db).
 
 ## Writing data through JSON + UDP
 
-InfluxDB allows you to write data through JSON and UDP. It assumes that you will be writing data to a single database, which is configured through the [configuration file](https://github.com/influxdb/influxdb/blob/master/config.sample.toml). The data you write in should look exactly like what you'd `POST` to the HTTP API.
+InfluxDB allows you to write data through JSON and UDP. It assumes that you will be writing data to a single database, which is configured through the [configuration file](https://github.com/influxdb/influxdb/blob/master/etc/config.sample.toml). The data you write in should look exactly like what you'd `POST` to the HTTP API.
```
Comment (otoolep) [inline source/docs/v0.8/api/reading_and_writing_data.md]: This change in paths only applies to upcoming 0.9.0 release.

**Comment by otoolep** [comment]
Thanks @bmillemathias -- we're actually overhauling the documentation right now, but this may be useful content. This PR actually has a mix of 0.8 and 0.9 content, so would need some changes. It might be best if you hold off for a little while until the 0.9.0 docs are available.

**Comment by bmillemathias** [comment]
AS the feature is already present in 0.8 I can provide a patch for this version

**Comment by otoolep** [comment]
OK @bmillemathias

**Comment by bmillemathias** [comment]
Branch updated to link to config.sample.toml for version 0.8.

**Comment by otoolep** [comment]
Thanks @bmillemathias

**Pull request merged**